.github/azure-workflows/main.yml

```
@@ -0,0 +1,175 @@
+trigger:
+  branches:
+    include:
+      - "update/*"
+
+pr:
+  - "*"
+
+
+resources:
+  repositories:
+    - repository: mpbb
+      type: github
+      endpoint: macports
+      name: macports/mpbb
+
+jobs:
+  - job: build
+    displayName: "macOS-10.14"
+    timeoutInMinutes: 0
+    pool:
+      vmImage: 'macOS-10.14'
+    steps:
+      - checkout: self
+        fetchDepth: 64
+        path: s/ports
+      - checkout: mpbb
+        path: s/mpbb
+      - bash: . ports/.github/azure-workflows/bootstrap.sh
+        displayName: Bootstrap MacPorts
+      - bash: |
+          set -eu
+          # Add getopt, mpbb and the MacPorts paths to $PATH for the subsequent
+          # steps.
+          echo "##vso[task.prependpath]/opt/mports/bin"
+          echo "##vso[task.prependpath]${PWD}/mpbb"
+          echo "##vso[task.prependpath]/opt/local/bin"
+          echo "##vso[task.prependpath]/opt/local/sbin"
+
+          IFS='/' read -r -a portlist <<<$( \
+            git -C ports/ diff --name-only --diff-filter=AM macports/master @ \
```

And right after I wrote the mail, I found the section in the docs. This needs to change to:
```suggestion
            git -C ports/ diff --name-only --diff-filter=AM macports/master...@ \
```